java/src/com/ibm/streamsx/topology/internal/streaminganalytics/VcapServices.java

```diff
@@ -22,82 +22,104 @@
 import com.google.gson.JsonParser;
 
 /**
- * Utilities to get the correct VCAP services
- * information for submission to Streaming Analytics Service.
+ * Utilities to get the correct VCAP services information for submission to
+ * Streaming Analytics Service.
  *
  */
 public class VcapServices {
-    
+
     /**
      * Get the top-level VCAP services object.
      * 
-     * Object can be:
-     * JsonObject - assumed to contain VCAP_SERVICES
-     * String -  assumed to contain serialized VCAP_SERVICES JSON
-     * File - assumed to be a file containing serialized VCAP_SERVICES JSON
+     * Object can be one of the following:
+     * <ul>
+     * <li>JsonObject - assumed to contain VCAP_SERVICES, and SERVICE_NAME</li>
```

Comment:
I'm confused by what this means, getVcapServices() just returns the vcap services without any setting of the service name to use.